cubi_tk/snappy/itransfer_step.py

```diff
@@ -0,0 +1,42 @@
+"""``cubi-tk snappy itransfer-step``: transfer step results into iRODS landing zone."""
+
+import argparse
+import os
+import typing
+
+from logzero import logger
+
+from .itransfer_common import SnappyItransferCommandBase
+
+
+class SnappyItransferStepCommand(SnappyItransferCommandBase):
+    """Implementation of snappy itransfer command for results from any step."""
+
+    fix_md5_files = True
+    command_name = "itransfer-step"
+    step_name = None
+
+    @classmethod
+    def setup_argparse(cls, parser: argparse.ArgumentParser) -> None:
+        super().setup_argparse(parser)
+        parser.add_argument(
+            "--step", help="Name of the snappy pipeline step", default="ngs_mapping"
```

Comment:
Shouldn't the default be `None`? I'd argue that it is better to fail than having users accidentality transferring ngs data.
You can probably also improve the help by giving examples. I never found obvious the relation between name of step and directory.